rdr_service/api/cloud_tasks_api.py

```diff
@@ -182,6 +185,30 @@ def post(self):
         return '{"success": "true"}'
 
 
+class PtscHealthDataTransferValidTaskApi(Resource):
+    """
+    Cloud Task endpoint: Ptsc Health Data Transfer Result
+    """
+    @task_auth_required
+    def post(self):
+        log_task_headers()
+        data = request.get_json(force=True)
+        logging.info(f'Ptsc Health Data Transfer Result: {data.get("attributes").get("eventType")}')
+        # possible event types: TRANSFER_OPERATION_SUCCESS, TRANSFER_OPERATION_FAILED, TRANSFER_OPERATION_ABORTED
+        event_type = data.get("attributes").get("eventType")
+        if event_type in ['TRANSFER_OPERATION_FAILED', 'TRANSFER_OPERATION_ABORTED']:
```

Comment:
This receives the status that is displayed in the UI for the data transfer jobs? Even when it successfully runs it still gets marked as a failure because of a few files that it doesn't have access to (temp files that don't appear to need transferred anyway). How should we handle those so we don't see failure messages for them?

Reply:
I saw those two files are failed due to no permission. I think we can either check PTSC for permission or request them to remove those two files if possible.